compute/hybrid/compute.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package hybridcompute
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	hybridnetwork "github.com/Azure-Samples/azure-sdk-for-go-samples/network/hybrid"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
```

Comment:
Why not also name this one `hybridcompute`? So it is very explicit it is the API version for hybrid

Reply:
Addressed